pv_site_api/_db_helpers.py

```diff
@@ -171,13 +175,16 @@ def get_generation_by_sites(
     session: Session, site_uuids: list[str], start_utc: dt.datetime
 ) -> list[MultiplePVActual]:
     """Get the generation since yesterday (midnight) for a list of sites."""
+    logger.info(f"Getting generation for {len(site_uuids)} sites")
     rows = get_pv_generation_by_sites(
         session=session, start_utc=start_utc, site_uuids=[uuid.UUID(su) for su in site_uuids]
     )
 
     # Go through the rows and split the data by site.
     pv_actual_values_per_site: dict[str, list[PVActualValue]] = defaultdict(list)
 
+    # TODO can we speed this up?
```

Comment:
We could *not* use pydantic for value we are returning, if that's the bottleneck, or see why pydantic is slow.

Reply:
lets find out the timings first, and then think.